apps/twitch-bot/internal/command/helpers/helpers_test.go

```diff
@@ -6,6 +7,87 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestParseMessage(t *testing.T) {
+	type TestCase struct {
+		description    string
+		input          string
+		expectedCmd    string
+		expectedParams []string
+	}
+
+	testCase := []TestCase{
+		{
+			description:    "with just message",
+			input:          "abc",
+			expectedCmd:    "",
+			expectedParams: nil,
+		},
+		{
+			description:    "with a command",
+			input:          "!kampus",
+			expectedCmd:    "kampus",
+			expectedParams: []string{},
+		},
+		{
+			description:    "with wrong mention",
+			input:          "@s",
+			expectedCmd:    "",
+			expectedParams: nil,
+		},
+		{
+			description:    "with just mention",
+			input:          "@senchabot",
+			expectedCmd:    "",
+			expectedParams: nil,
+		},
+		{
+			description:    "a command and its params",
+			input:          "!acmd acommand a command content",
+			expectedCmd:    "acmd",
+			expectedParams: []string{"acommand", "a", "command", "content"},
+		},
+		{
+			description:    "if there is space in params, do not include the space in the params",
+			input:          "!acmd a couple of  params",
```

Comment:
This looks like a duplicate of the test: "a command and its params"

Reply:
The above is the correct usage, but this one is the incorrect usage, but since they both expect the same parameters, the other one can probably be removed